tests/clone_test.go

```diff
@@ -869,3 +849,23 @@ func stopCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (
 
 	return virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
 }
+
+func withFirmware(firmware *virtv1.Firmware) libvmi.Option {
+	return func(vmi *virtv1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Firmware = firmware
+	}
+}
+
+func createSourceVM(options ...libvmi.Option) *virtv1.VirtualMachine {
+	vmi := libvmifact.NewCirros(options...)
+	vmi.Namespace = testsuite.GetTestNamespace(nil)
+	vm := libvmi.NewVirtualMachine(vmi,
+		libvmi.WithAnnotations(vmi.Annotations),
+		libvmi.WithLabels(vmi.Labels))
+
+	By(fmt.Sprintf("Creating VM %s", vm.Name))
+	virtClient := kubevirt.Client()
```

Comment:
nit: I think this variable substitution is redundant, `kubevirt.Client()` can be called directly

Reply:
prefer to keep.